examples/assign_interface_ip_address_manual/assign_interface_ip_address_manual.yaml

```diff
@@ -0,0 +1,18 @@
+- name: Assign interface IP address
+  connection: local
+  hosts: localhost
+  tasks:
+  - name: Assign interface IP address
+    a10_interface_ethernet:
+      state: "{{ state }}"
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      ip:
+        address_list:
+        -  ipv4-address: "{{ ipv4-address }}"
```

Comment:
Switch this to some example data instead of a variable.

Reply:
Hi @hthompson6 : please check the changes